### PR TITLE
docs: editorial suggestions to new C/C++ topics

### PR DESCRIPTION
**Pull request by jf205**

### CODE REVIEW

File: docs/language/learn-ql/cpp/value-numbering-hash-cons.rst [hunk 1]
```diff
@@ -55,6 +59,7 @@ To get the ``GVN`` of an ``Expr``, use the ``globalValueNumber`` predicate.
 
 Why not a predicate?
 ~~~~~~~~~~~~~~~~~~~~
+
 The obvious interface for this library would be a predicate ``equivalent(Expr e1, Expr e2)``. However, this predicate would be very large, with a quadratic number of rows for each set of equivalent expressions. By using a class as an intermediate step, the number of rows can be kept linear, and therefore can be cached.
 
 Example Queries
```
Comment (shati-patel) [inline docs/language/learn-ql/cpp/value-numbering-hash-cons.rst]:
```suggestion
Example queries
```

(Just spotted this when scrolling through.)

File: docs/language/learn-ql/cpp/value-numbering-hash-cons.rst [hunk 2]
```diff
@@ -55,9 +59,10 @@ To get the ``GVN`` of an ``Expr``, use the ``globalValueNumber`` predicate.
 
 Why not a predicate?
 ~~~~~~~~~~~~~~~~~~~~
+
 The obvious interface for this library would be a predicate ``equivalent(Expr e1, Expr e2)``. However, this predicate would be very large, with a quadratic number of rows for each set of equivalent expressions. By using a class as an intermediate step, the number of rows can be kept linear, and therefore can be cached.
 
-Example Queries
+ Example queries
```
Comment (shati-patel) [inline docs/language/learn-ql/cpp/value-numbering-hash-cons.rst]: Sorry, on second thought, there's only one example... Perhaps change it to "Example query" or just "Example". Ditto for [line 98](https://github.com/Semmle/ql/pull/2160/files#diff-6368dc543c8e0bece775dbc865690c5eR98) of this file.

Comment (jf205) [reply docs/language/learn-ql/cpp/value-numbering-hash-cons.rst]: I happy with both of those suggestions! Feel free to add them (if possible).

Comment (shati-patel) [reply docs/language/learn-ql/cpp/value-numbering-hash-cons.rst]: I can't edit line 98, since it isn't in the diff :(

**Comment by rdmarsh2** [comment]
LGTm

**Pull request merged**